<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PcApp</class>
 <widget class="QMainWindow" name="PcApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PcApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>551</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bLoadData">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Try to load data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSetMaxV">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set max V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSearchFile">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>230</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Search data file</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>811</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Unknow state, please try to connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bConnectAr">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Try to Connect </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="voltageInp">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="percentageInp">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bSetMin">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Min V</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
